frontend/app/dashboard/src/views/dashboard/settings/PremiseView.vue

```diff
@@ -0,0 +1,176 @@
+<template>
+    <SaveView :title="title" :loading="saving" :disabled="!hasChanges" @save="save" v-on="!isNew && deleteHandler ? {delete: deleteMe} : {}">
+        <div class="container">
+            <h1>{{ title }}</h1>
+            <hr>
+            <div class="container">
+                <h2>Adres</h2>
+                <AddressInput v-model="address" title="" :validator="errors.validator" :link-country-to-locale="true" />
+            </div>
```

Comment:
```suggestion
            <h2>Adres</h2>
            <AddressInput v-model="address" title="" :validator="errors.validator" :link-country-to-locale="true" />
```

Containers zijn eerder voor conditionele stukken, om templates te vermijden

Reply:
`<hr>`altijd in de containers zetten ook